### PR TITLE
Add changelog for 1.2.0b1.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
@maciejmrowiec: Some server components that have been updated don't have a single changelog entry. Should we add something?

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 1]
```diff
@@ -4,6 +4,66 @@ taxonomy:
...
 ---
 
+## v1.2.0 Beta
+
+_Released xx.xx.xxxx_
+
+#### deployments
+* Deployment creation process changed. From now on artifacts are
+  assigned to device deployments on update request handling.
+
```
Comment (maciejmrowiec) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: * Return 422 - Unprocessable Entity on attempt of creating deployment without artifacts
* Deployments no longer require inventory to create deployments.
* New optional array field: 'artifacts' in deployment object retuned by API containing list of artifact ids used by deployment.

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 2]
```diff
@@ -4,6 +4,66 @@ taxonomy:
...
+#### deviceauth
```
Comment (maciejmrowiec) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: * Introduce 'server' subcommand that is also default command. Supports '--automigrate' parameter to enable automatic database version migration on startup.
* Increase orchestrator request timeout to 30s

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 3]
```diff
@@ -4,6 +4,66 @@ taxonomy:
...
+  (MEN-1178)
+* Client will not run state scripts from cmd-line except when forced.
+  (MEN-1235)
+* installer: improve incompatible image error message
+* Refactored all store implementations into /store
+* Introduction of state script feature. State scripts can be
+  used to execute scripts at various stages of Mender's execution. See
+  documentation for more information.
+
+#### mender-api-gateway-docker
```
Comment (maciejmrowiec) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: * Return additional headers for improved security: X-XSS-Protection, Cache-Control, Pragma. (MEN-1316)
* Validate Origin header if present. (MEN-1287)
* Add a configurable Host whitelist to gateway configuration, denying requests with unknown Hosts. Configured through ALLOWED_HOSTS env var on gateway startup. (MEN-1262)

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 4]
```diff
@@ -4,6 +4,66 @@ taxonomy:
...
+
+#### mender-artifact
+* Fix misleading version being displayed for non-tagged builds.
+  (MEN-1178)
+* Improve error message when private signing key can't be loaded.
+* Sign existing artifacts using mender-artifact CLI (MEN-1220)
+
+#### useradm
```
Comment (maciejmrowiec) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: * Remove 'initial user' login logic, including 'POST /users/initial' API. Now initial user need to be created by administrator using cli (MEN-1034)
* New cli subcommand for creating users: 'useradm create-user. (MEN-1034)
* New API for listing users: 'GET https://localhost/api/management/v1/useradm/users' and 'GET https://localhost/api/management/v1/useradm/users/:userid'
* New API for creating additional users: 'POST https://localhost/api/management/v1/useradm/users'
* New API for editing user email and password: 'PUT https://localhost/api/management/v1/useradm/users/:userid'
* New API for removing user: 'DELETE https://localhost/api/management/v1/useradm/users/:userid'

**Pull request merged**